src/mono/wasm/Makefile

```
@@ -111,7 +111,10 @@ $(BUILDS_OBJ_DIR)/corebindings.o: runtime/corebindings.c | $(BUILDS_OBJ_DIR)
 $(NATIVE_BIN_DIR)/src/emcc-flags.txt: | $(NATIVE_BIN_DIR)/src
 	echo "$(call escape_quote,$(EMCC_FLAGS)) $(1) -Oz" > $$@
 
-build-native: $(NATIVE_BIN_DIR)/dotnet.js $(NATIVE_BIN_DIR)/src/emcc-flags.txt
+$(NATIVE_BIN_DIR)/src/emcc-version.txt: | $(NATIVE_BIN_DIR)/src
```

add a dependency on the emsdk version so we regenerate this file when it changes:
```suggestion
$(NATIVE_BIN_DIR)/src/emcc-version.txt: $(EMSDK_PATH)/upstream/.emsdk_version | $(NATIVE_BIN_DIR)/src
```